modules/calc/src/main/java/com/opengamma/strata/calc/runner/function/result/FxConvertibleList.java

```diff
@@ -63,8 +67,8 @@
    * @return an instance with the specified values
    */
   @SafeVarargs
-  public static FxConvertibleList of(FxConvertible<?>... values) {
-    return new FxConvertibleList(ImmutableList.copyOf(values));
+  public static <R extends FxConvertible<?>> FxConvertibleList<R> of(R... values) {
```

Comment:
In other places we've tended to use the same type parameter name in static methods as in the class, i.e. `T` in this case.